samples/sample_multi_transcode/src/sample_multi_transcode.cpp

```diff
@@ -596,8 +598,9 @@ mfxStatus Launcher::ProcessResult()
            << MSDK_STRING(" [") << m_pThreadContextArray[i]->pPipeline->GetSessionText()
            << MSDK_STRING("] ") << SessionStsStr <<MSDK_STRING(" (")
            << StatusToString(_sts) << MSDK_STRING(") ")
-           << m_pThreadContextArray[i]->working_time << MSDK_STRING(" sec, ")
-           << m_pThreadContextArray[i]->numTransFrames << MSDK_STRING(" frames")
+           << _workTime << MSDK_STRING(" sec, ")
+           << _framesNum << MSDK_STRING(" frames, ") 
+           << std::fixed << std::setprecision(3) << _framesNum / _workTime << MSDK_STRING(" fps")
```

Comment:
It won't work, ```_framesNum``` is integer, so result of ```_framesNum / _workTime``` is integer too. Fix it by casting: ```mfxF64(_framesNum) / _workTime```

Reply:
Why? _workTime already float value. If either the divisor or the dividend is a floating-point number, a floating-point divide is executed.
Also I checked it before (got float) :)

Reply:
Oh, my bad, I missed that ```workTime``` is ```float```